### DISCUSSION TITLE
til-site/posts/tmux-is-worse-is-better/

### DISCUSSION THREAD

**Discussion by giscus** [hiAndrewQuinn/til#15] [Announcements]
# til-site/posts/tmux-is-worse-is-better/

tmux (short for “terminal mux” (short for “multiplexer”)) is i3 for your terminal. Oh, it’s so much more than that, and I recently discovered with some joy that it is installed by default on OpenBSD, but its fundamental value add to any programmer who has to SSH into servers more than once a week is it allows you to split your screen up into multiple independent shells without needing a graphical environment at all.

https://hiandrewquinn.github.io/til-site/posts/tmux-is-worse-is-better/?utm_source=hackernewsletter&utm_medium=email&utm_term=cutting_room_floor

**Comment by ziogas** [comment]
I've solved the kitty-tmux muscle memory problem by adding tmux-like shortcuts to kitty:

```
# tmux-like key bindings
map ctrl+b>left neighboring_window left
map ctrl+b>right neighboring_window right
map ctrl+b>up neighboring_window up
map ctrl+b>down neighboring_window down
map ctrl+b>] next_tab
map ctrl+b>[ previous_tab
map ctrl+b>n next_tab
map ctrl+b>p previous_tab
map ctrl+b>" launch --location=vsplit --cwd=current --type=window
map ctrl+b>% launch --location=hsplit --cwd=current --type=window
map ctrl+b>space next_layout
map ctrl+b>z toggle_layout stack
map ctrl+b>x close_window
map ctrl+b>c launch --cwd=current --type=tab
```

you can tweak them to work the same way as tmux does.

of course, it doesn't solve the problem if you want to open a few TTYs on remote host without ssh'ing multiple times 🙂
